pkg/agent/datamodel/sig_config.go

```diff
@@ -517,20 +551,28 @@ func GetSIGAzureCloudSpecConfig(sigConfig SIGConfig, region string) (SIGAzureEnv
 		return SIGAzureEnvironmentSpecConfig{}, fmt.Errorf("unexpected error while constructing env-aware sig configuration for Windows: %s", err)
 	}
 	c.SigWindowsImageConfig = getSigWindowsImageConfigMapWithOpts(fromACSWindows)
+
+	fromACSUbuntuEdgeZone, err := withEdgeZoneConfig(sigConfig)
```

Comment:
you could have hardcoded all the RG/gallery names in AB, and only consumed the subscription id from RP, which would allow some refactoring, but this is fine

Reply:
I wanted to do that initially, but decided to go with the minimal working refactor, can refactor those things out later on.